styles.css

```diff
@@ -0,0 +1,328 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+
+* {
+  margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+  box-sizing: border-box;
+}
```

Comment:
Please use 2 spaces indentation

Reply:
- [ ] I think it would be better here also to mention the reason behind this request because may be it tis obvious to you but not to others 🔥✅🚀